Enable commands for display and manipulation of UEFI boot options by
default.

Signed-off-by: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
---
 cmd/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cmd/Kconfig b/cmd/Kconfig
index 02c298fdbe..cfc4563f59 100644
--- a/cmd/Kconfig
+++ b/cmd/Kconfig
@@ -502,6 +502,7 @@ config CMD_ENV_FLAGS
 config CMD_NVEDIT_EFI
 	bool "env [set|print] -e - set/print UEFI variables"
 	depends on EFI_LOADER
+	default y
 	imply HEXDUMP
 	help
 	  UEFI variables are encoded as some form of U-Boot variables.
@@ -1757,6 +1758,7 @@ config CMD_CLS
 config CMD_EFIDEBUG
 	bool "efidebug - display/configure UEFI environment"
 	depends on EFI_LOADER
+	default y
 	select EFI_DEVICE_PATH_TO_TEXT
 	help
 	  Enable the 'efidebug' command which provides a subset of UEFI
-- 
2.33.1

